### PR TITLE
Fixes #12 ArrayUnion

### LINKED ISSUE #12: Lab 3 - Arrays Union
Let's say you have two arrays:
  ```
a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
b = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
```

Write a program that returns a union between these arrays (without duplicates).

How it looks?
```
Please input the first array:
>>> 1 1 2 3 5 8 13 21 34 66 89
Please input the second array:
>>> 1 2 3 4 5 6 7 8 9 10 11 12 13
Distinct union is: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 21, 34, 55, 89
```

### PR DESCRIPTION
**Pull request by ArtemDomnikoff**

### CODE REVIEW

File: 4-Alphabet/Alphabet.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+def printSymbols(columns, type, alphabetRangeStart, alphabetRangeEnd):
```
Comment (daria-gh) [inline 4-Alphabet/Alphabet.py]: Please remove this file from the request

File: 5-Array-Union/Array Union.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+def array_union(first, second):
+    result = []
+    try:
+        if first != ['']:
+            first = contains_2(first)
+            for number in first:
+                cond = contains(number, second)
```
Comment (daria-gh) [inline 5-Array-Union/Array Union.py]: Use directly  ```if number in array:``` here
The function ```def contains(number, array):``` doesn't have any logic

File: 5-Array-Union/Array Union.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+def array_union(first, second):
+    result = []
+    try:
+        if first != ['']:
+            first = contains_2(first)
```
Comment (daria-gh) [inline 5-Array-Union/Array Union.py]: Rename to ``contains```

File: 5-Array-Union/Array Union.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    except ValueError:
+        if number in first:
+            first = delete_element(first, number)
+        elif number in second:
+            second = delete_element(second, number)
+        result = array_union(first, second)
+        return result
+
+
+def delete_element(array, element):
```
Comment (daria-gh) [inline 5-Array-Union/Array Union.py]: Remove this function. It do nothing except ```array.remove(element)```. No reason to have a separate function for it.

**Review by daria-larionova — CHANGES_REQUESTED** [review]
Please see my comments

**Pull request merged**